pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -359,6 +359,8 @@ func NewControllerDeployment(namespace string, repository string, imagePrefix st
 		VirtControllerName,
 		"--launcher-image",
 		fmt.Sprintf("%s/%s%s%s", repository, imagePrefix, "virt-launcher", launcherVersion),
+		"--exporter-image",
+		"ghcr.io/mhenriks/virt-exportserver:latest",
```

Comment:
@mhenriks Is working on the exporter pod in parallell, and for now he doesn't have a PR for it yet (since he is waiting for this PR). But his work is available here: https://github.com/mhenriks/kubevirt/tree/exportserver which will be come a kubevirt some time in the near future.

Reply:
Thanks for explaining

Reply:
I don't think we can accept this because of the use of the `latest` tag. If we're going to functionally test this feature, we have to lock it into an exact container, or else we risk blocking PRs from being merged if this container tag mutates and breaks the tests.